errors/list.go

```diff
@@ -104,23 +111,38 @@ func (l *List) Detailed() string {
 //
 // Any error of type errors.List will be flattened inside
 // the error list.
-func (l *List) Append(err error) {
-	if err == nil {
+func (l *List) Append(errs ...error) {
```

Comment:
Kinda curious when we append multiple errors, we usually call something that returns an error instance and then already call Append on it 🤔

Reply:
This is not actually needed anymore but I think it makes sense to support this.

1. it's intuitive, in the same way we can do `errs = append(errs, err1, err2, ...)`
2. when wrapping multiple lists of errors with different kinds.

```
    el1 := errors.L()
    for ... {
        err = ...
        el1.Append(err)
        ...
    }
    
    el2 := errors.L()
    for ... {
        err = ...
        el2.Append(err)
    }
    
    allerrs := errors.L()
    allerrs.Append(errors.E(SuchFail, el1), errors.E(SuchFail, el2))
    return allerrs.AsError()
```

but current code doesn't depend on this anymore so I can rollback to single parameter if you do prefer.

Reply:
Hmm it doesn't change the usage of the method so I suppose it is OK to leave with vargs since it can be useful on some scenarios, was just curious when that happens because I usually just keep appending errors as soon as they happen etc.